bentoml/_internal/io_descriptors/json.py

```diff
@@ -123,30 +124,31 @@ def __init__(
         self._validate_json = validate_json
         self._json_encoder = json_encoder
 
-    def openapi_schema(self) -> t.Dict[str, t.Dict[str, t.Dict[str, t.Any]]]:
+    def openapi_schema_type(self) -> t.Dict[str, t.Any]:
         if hasattr(self, "_pydantic_model"):
-            schema = self._pydantic_model.schema()
-        else:
-            schema = {"type": "object"}
-        return {MIME_TYPE_JSON: {"schema": schema}}
+            return self._pydantic_model.schema()
+        return {"type": "object"}
 
     def openapi_request_schema(self) -> t.Dict[str, t.Any]:
         """Returns OpenAPI schema for incoming requests"""
-        return self.openapi_schema()
+        return {MIME_TYPE_JSON: {"schema": self.openapi_schema_type()}}
 
     def openapi_responses_schema(self) -> t.Dict[str, t.Any]:
         """Returns OpenAPI schema for outcoming responses"""
-        return self.openapi_schema()
+        return {MIME_TYPE_JSON: {"schema": self.openapi_schema_type()}}
 
     async def from_http_request(self, request: Request) -> JSONType:
         json_obj = await request.json()
         if hasattr(self, "_pydantic_model") and self._validate_json:
+            if pydantic is None:
+                raise MissingDependencyException(
+                    "`pydantic` must be installed to use `pydantic_model`"
+                )
             try:
-                return self._pydantic_model.parse_obj(json_obj)
+                self._pydantic_model.parse_obj(json_obj)
```

Comment:
So just verifying but not regulating the json_obj right?

Reply:
`parse_obj` actually return a `pydantic.BaseModel` instead of a JSON object. I think we should only use pydantic for validation

Reply:
maybe we should consider returning the pydantic obj instead of json_obj here? or make that an option?  let's have a follow up discussion

Reply:
just discussed with @bojiang, we all agree that it should just return the pydantic object in this case, cc @aarnphm